liri.js

```diff
@@ -0,0 +1,173 @@
+//npm packages...
+var fs = require("fs");
+
+var keys = require("./keys.js");
+
+var request = require("request");
+
+var Twitter = require('twitter');
+
+var Spotify = require('node-spotify-api');
+
+var fs = require("fs");
+
+var inquirer = require("inquirer");
+
+// Created a series of questions that will eventually determine what function to call and what data to pass into it.
+inquirer.prompt([
+
+  {
+    type: "input",
+    name: "name",
+    message: "What is your name?"
+  },
+
+  {
+    type: "list",
+    name: "opperator",
+    message: "What would you like me to do for you?",
+    choices: ["Show your tweets", "Spotify a song", "IMDB a movie", "Do what ever it says"]
+  }
+
+]).then(function(answers) {
+
+  if (answers.opperator == "Spotify a song") {
+    inquirer.prompt([
+      {
+        type: "input",
+        name: "songName",
+        message: "What song?"
+      }
+    ]).then(function(answers) {
+      songName = answers.songName;
+      spotify();
```

Comment:
I would construct this in a way such that your spotify function takes answers.songName as an argument and not interpreting it as a global variable.